examples/js/renderers/CSS3DRenderer.js

```diff
@@ -13,11 +13,15 @@ THREE.CSS3DObject = function ( element ) {
 
 	this.addEventListener( 'removed', function () {
 
-		if ( this.element.parentNode !== null ) {
+		this.traverse( function( object ) {
 
-			this.element.parentNode.removeChild( this.element );
+			if ( object.element !== null && object.element.parentNode !== null ) {
```

Comment:
`element` won't be null and should be `instanceof Element`, because of https://github.com/mrdoob/three.js/blob/5bf9029a0416ddf739582f998c507c3bfc7602a4/examples/js/renderers/CSS3DRenderer.js#L12
Otherwise, it shows an error at initializing.

I think `object.element !== null` can be omitted

Reply:
sorry, forget about above, `CSS3DObject` may contain `Object3D`.

maybe, 
```suggestion
			if ( object.element instanceof Element  && object.element.parentNode !== null ) {
```

Reply:
But the child can be just an Object3D e.g. a Group